### DISCUSSION TITLE
How to "run" an application that is just a script or set of scripts? But doesn't actually run a container?

### DISCUSSION THREAD

**Discussion by douglasg14b** [coollabsio/coolify#4613] [Q&A]
I have a need to run misc scripts on intervals for different jobs/tasks. Think of it like something similar to a github workflow that run on a cron.

I can't seem to figure out how to only have a resource that doesn't want ports, a docker container, builds...etc It's just repo that has commands which run scripts with node.

How can I do this?

**Comment by itsjavi** [comment]
It would be nice to have.

My use case: I have a repo with Dockerfile(s) that I want to use as base images of my projects, and when I push to main, I'd like to coolify to just run my script to deploy the images to my docker registry, without needing to start a container.

Running this kind of scripts via scheduled tasks is not convenient at all
